src/test/java/business/CommonSteps.java

```diff
@@ -0,0 +1,18 @@
+package business;
+
+import core.Config;
+import org.testng.annotations.Test;
+
+import static core.Config.logger;
+
```

Comment:
Please use more specific names for class. Since it contains @Test annotation it is obvious to have Test in class name

Reply:
fixed